### ISSUE TITLE
`RETURNING`  produces wrong column name

### ISSUE THREAD

**Issue by mgirlich** [r-dbi/RSQLite#381]
While working on the support for `RETURNING` in `dm::rows_insert()` https://github.com/cynkra/dm/issues/593 I noticed that the column names produced by returning are escaped

``` r
library(RSQLite)

x <- tibble::tibble(
  x = 1:3,
  y = runif(3)
)

con <- DBI::dbConnect(RSQLite::SQLite(), ":memory:", create = TRUE)
DBI::dbWriteTable(con, "tbl", x, overwrite = TRUE)
DBI::dbWriteTable(con, "tbl2", x, overwrite = TRUE)

DBI::dbGetQuery(
  con,
  "INSERT INTO tbl (`x`, `y`)
   SELECT * FROM tbl2
   RETURNING `x`",
  immediate = TRUE
)
#>   `x`
#> 1   1
#> 2   2
#> 3   3

DBI::dbGetQuery(
  con,
  "SELECT `x` FROM tbl2",
  immediate = TRUE
)
#>   x
#> 1 1
#> 2 2
#> 3 3
```

<sup>Created on 2021-08-17 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by mgirlich** [comment]
This was also fixed in SQLite 3.36.0 🥳

**Comment by krlmlr** [comment]
I'll submit to CRAN today.

How can we ensure that we use a very recent version of RSQLite in dm?
